### ISSUE TITLE
Which wav2vec2 model should I choose for reproducing SUPERB?

### ISSUE THREAD

**Issue by YangHao97** [s3prl/s3prl#271]
Hi, I'm trying to implement the results of wav2vec 2.0 Large and wav2vec 2.0 Base on the superb leaderboard. I would like to know which parameters are right, -u wav2vec2_large_960 and -u wav2vec2_base_960 ?

**Comment by leo19941227** [comment]
Hi,

Please use `-u wav2vec2` for the base model and `-u wav2vec2_large_ll60k` for the large model.
If you wish to know the best finetuning config we found in the paper, please refer to issue #268.

**Comment by YangHao97** [comment]
Thanks!

**Comment by YangHao97** [comment]
Hi, another question, in order to reproduce SUPERB, should I use training mode or fine-tuning mode?

**Comment by leo19941227** [comment]
You should use `-m train` without `-f`. So, only the downstream model will be trained and the pretrained model (upstream) is frozen.

**Comment by YangHao97** [comment]
Thank you!

**Comment by Arthurwyf** [comment]
Hello, if i want to pretraining wav2vec? What should I do
